12/task1_phonebook.py

```diff
@@ -0,0 +1,140 @@
+"""
```

Comment:
General comment to this solution: it would be much better to load data from a file once you start this program and write the content to a `dict`. Then, you can work with dict all the time and rewrite the file only in case of write operations: `add` and `delete`. Everything else could be done by accessing `dict`

Reply:
But if other users will have access to the DB file and will modify data some how - we will have 2 different dicts.

Reply:
It makes sense from your point of view, but let's assume we change our file only using this program.